data.lua

```diff
@@ -11,6 +11,9 @@ data:extend(
             alt_selection_mode = {"any-entity"},
             selection_cursor_box_type = "entity",
             alt_selection_cursor_box_type = "entity",
+      		entity_type_filters = {"resource"},
+      		alt_entity_type_filters = {"resource"},
+            flags = {"only-in-cursor", "spawnable"},

```

Comment:
Mind the whitespace :)

```suggestion
            entity_type_filters = {"resource"},
            alt_entity_type_filters = {"resource"},
            flags = {"only-in-cursor", "spawnable"},

```

Reply:
Yeah, sorry for the mess. The incorrect whitespace is probably due to the editor I used but it wasn’t my top priority either. I had some trouble with inexplicable conflicts between the different branches I created and the local and online versions. I am not that experienced with GitHub.